client/src/components/QandAList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import QandAItem from './QandAItem.jsx';
+
+
+const QandAlist = ( {questionsAndAnswers} ) => (
+
+  <div className='q-and-a-list' data-testid='QAndAList'>
+    {console.log('qandA', questionsAndAnswers)}
+    <QandAItem questionsAndAnswers={questionsAndAnswers}/>
+  </div>
+);
```

Comment:
I'm not sure I understand the purpose of this component's existence. If it is just passing down the same props to a different component within a single div, then it could just as easily be a div within `QuestionsAndAnswers`. Is there something I'm missing? Or are there any future plans for this?